server/files/stylesheets/docs.css

```diff
@@ -38,7 +38,16 @@ p:last-child {
 footer.footer {
   padding: 4em 1em !important;
 }
-
+.footer .ui.link.list > .item svg {
+  width: 0.7em;
+  height: auto;
+  margin-right: 0.30em;
```

Comment:
0.3em should be sufficient, no trailing zeros needed

Reply:
Okay I going to change now